packages/jest-haste-map/src/ModuleMap.ts

```diff
@@ -84,12 +89,15 @@ export default class ModuleMap {
   }
 
   toJSON(): SerializableModuleMap {
-    return {
-      duplicates: Array.from(this._raw.duplicates),
-      map: Array.from(this._raw.map),
-      mocks: Array.from(this._raw.mocks),
-      rootDir: this._raw.rootDir,
-    };
+    if (!this.json) {
```

Comment:
Just to verify, this class is re-instantiated once the data changes, so caching it is safe?

Reply:
Correct.